### ISSUE TITLE
Period at the end of Linked Message keys

### ISSUE THREAD

**Issue by eXistentialMisanthropy** [kazupon/vue-i18n#246]
Hello,

It seems that when you want to use a Linked Message Key just before the end of a sentence, the Key instead of the Value would be rendered. This is because of the Message Syntax that allows nested JSON in the resource files. e.g. messages.name

`ret.match(/(@:[\w\-_|.]+)/g)` found in `index.js`.This regEx incorporates hyphens and underscores as suggested by older Issues, but also incorporates the period character in order to maintain nested JSON support.

A suggestion is to provide an ending delimiter to the "@:" format. Welcome to have any other solutions!

Thank You!

**vue & vue-i18n version**
```
"vue": "^2.4.4",
"vue-i18n": "^7.3.2"
```

**Steps to reproduce**
`enca.json`
```
{
    "introMessage": "Welcome @:firstName, Thank you for considering the @:desiredProduct. We are here to help!",
    "firstName": "Elon",
    "desiredProduct": "Solar Roof"
}
```

**What is Expected?**
`{{ $t("introMessage") }}` should render 'Welcome Elon, Thank you for considering the Solar Roof. We are here to help!'

**What is actually happening?**
`{{ $t("introMessage") }}` renders 'Welcome Elon, Thank you for considering the desiredProduct. We are here to help!'

**Comment by nelson6e65** [comment]
This can maybe be solved like in other langs, like Bash o PHP, by using `@:{...}` or similar:

```js
"Welcome @:{firstName}, Thank you for considering the @:{desiredProduct}. We are here to help!"
```


----
By the way (maybe related), I use some keys as strings with spaces (like `'Sign up`', `'Basic user'`, etc), and is not possible to link them. :c

**Comment by exoego** [comment]
@eXistentialMisanthropy @nelson6e65 
This was finally implemented in v8.2.0.
https://kazupon.github.io/vue-i18n/guide/messages.html#grouping-by-brackets

**Comment by eXistentialMisanthropy** [comment]
Thanks!
